src/sentry/snuba/metrics/extraction.py

```diff
@@ -591,16 +603,17 @@ def failure_tag_spec(_1: Project, _2: Optional[str]) -> List[TagSpec]:
     ]
 
 
-def apdex_tag_spec(project: Project, argument: Optional[str]) -> List[TagSpec]:
+def apdex_tag_spec(project: Project, arguments: Optional[List[str]]) -> List[TagSpec]:
     _, metric = _get_apdex_project_transaction_threshold(project)
 
     # TODO: we can also opt to fallback on the db threshold in case it's not supplied, but we have to see if we want to
     #  support that.
-    if argument is None:
+    if arguments is None:
```

Comment:
Covers both `None` and empty list.

```suggestion
    if not arguments:
```